public/css/main.css

```diff
@@ -33,4 +33,8 @@ h1 {
 
 .u-bgc--f4f4f4 {
   background-color: #f4f4f4;
+}
+
+.ph2-ns {
+  padding: 20vh 0 0 0;
```

Comment:
I don't think we should use the same class name. If we can't use some other padding class (https://github.com/tachyons-css/tachyons-spacing) then we should add a custom class.